libraries/entities/src/EntityItem.cpp

```diff
@@ -339,18 +338,18 @@ OctreeElement::AppendState EntityItem::appendEntityData(OctreePacketData* packet
         APPEND_ENTITY_PROPERTY(PROP_SERVER_SCRIPTS, getServerScripts());
 
         // Certifiable Properties
-        APPEND_ENTITY_PROPERTY(PROP_ITEM_NAME, getItemName());
-        APPEND_ENTITY_PROPERTY(PROP_ITEM_DESCRIPTION, getItemDescription());
-        APPEND_ENTITY_PROPERTY(PROP_ITEM_CATEGORIES, getItemCategories());
-        APPEND_ENTITY_PROPERTY(PROP_ITEM_ARTIST, getItemArtist());
-        APPEND_ENTITY_PROPERTY(PROP_ITEM_LICENSE, getItemLicense());
-        APPEND_ENTITY_PROPERTY(PROP_LIMITED_RUN, getLimitedRun());
-        APPEND_ENTITY_PROPERTY(PROP_MARKETPLACE_ID, getMarketplaceID());
-        APPEND_ENTITY_PROPERTY(PROP_EDITION_NUMBER, getEditionNumber());
-        APPEND_ENTITY_PROPERTY(PROP_ENTITY_INSTANCE_NUMBER, getEntityInstanceNumber());
-        APPEND_ENTITY_PROPERTY(PROP_CERTIFICATE_ID, getCertificateID());
-        APPEND_ENTITY_PROPERTY(PROP_CERTIFICATE_TYPE, getCertificateType());
-        APPEND_ENTITY_PROPERTY(PROP_STATIC_CERTIFICATE_VERSION, getStaticCertificateVersion());
+        APPEND_ENTITY_PROPERTY(PROP_ITEM_NAME, QString());
+        APPEND_ENTITY_PROPERTY(PROP_ITEM_DESCRIPTION, QString());
+        APPEND_ENTITY_PROPERTY(PROP_ITEM_CATEGORIES, QString());
+        APPEND_ENTITY_PROPERTY(PROP_ITEM_ARTIST, QString());
+        APPEND_ENTITY_PROPERTY(PROP_ITEM_LICENSE, QString());
+        APPEND_ENTITY_PROPERTY(PROP_LIMITED_RUN, quint32(-1));
+        APPEND_ENTITY_PROPERTY(PROP_MARKETPLACE_ID, QString());
```

Comment:
Maybe remove this one and those below? Name, description, categories, artist, license are pretty much certain to come handy in the future, but the rest look like hifi marketplace specific values.

Reply:
These properties are specific to the marketplace.  I've done this so as to not break protocol until we're ready to, but if we keep them in this PR a later PR would likely drop these lines.